Replicator/ReplicatorOptions.hh

```diff
@@ -259,14 +251,34 @@ namespace litecore { namespace repl {
             return collectionOpts.size();
         }
 
-        Mode push(unsigned i) const {
+        Mode push(CollectionIndex i) const {
             return collectionOpts[i].push;
         }
 
-        Mode pull(unsigned i) const {
+        Mode pull(CollectionIndex i) const {
             return collectionOpts[i].pull;
         }
 
+        Validator pushFilter(CollectionIndex i) const {
+            return collectionOpts[i].pushFilter;
+        }
+
+        Validator pullValidator(CollectionIndex i) const  {
```

Comment:
Should be pullFilter(CollectionIndex i) for consistency.

Reply:
yes, I could use pullFilter, but then I need to change all current code to use pullFilter instead of pullValidator.

Reply:
Yes, there are 5 places in IncomingRev.cc that uses this functions based on your PR.

Reply:
I have changed them. I think better be consistent